### PR TITLE
For commenting on 01-08-Overscan.ipynb (part of review-ee0cbc6)

### PR DESCRIPTION
**Pull request by mwcraig**

### CODE REVIEW

File: notebooks/01-08-Overscan.ipynb [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+   "source": [
+    "The overscan region of a CCD, if present, is a part of the chip that is covered.\n",
+    "Depending on the camera, it can be a useful way to remove small variations in\n",
+    "the bias level from frame to frame.\n",
+    "\n",
+    "However, whether or not the overscan is useful depends on the camera. It's\n",
+    "advisable to examine the overscan part of the camera you're using before\n",
+    "deciding if you should include it in image reduction.\n",
+    "\n",
+    "One important note: *overscan always includes bias, read noise, and dark\n",
```
Comment (StuartLittlefair) [inline notebooks/01-08-Overscan.ipynb]: Hi Matt

This is potentially misleading. Overscan can be virtual or physical. You’ve described physical overscan here, but sometimes overscan is provided by simply operating more clocking cycles than there are rows in the CCD; in this case the overscan region does NOT contain dark current. (I think you see exactly this in your dark current image from a peltier cooled camera; the overscan and main image have different levels due to dark current). 

There’s also prescan, which is where the amplifier is clocked without clocking the CCD rows, giving a virtual set of rows that emerge before the CCD days that contain only bias and read noise.

Comment (mwcraig) [reply notebooks/01-08-Overscan.ipynb]: Hi Stuart,

Thanks for the detailed comment! I had been under the mistaken impression that overscan was always physical. 

According to the chip manufacturer for the peltier cooled camera the overscan should be physical in that case. However, it also should be possible to subframe as small a region as one wants (e.g. 64x64) and it definitely is not.

The other thing I find a little confusing is that counts are higher in the overscan that the main sensor for bias and dark, but not for science images. 🤷‍♂

Comment (StuartLittlefair) [reply notebooks/01-08-Overscan.ipynb]: Ha! The difference between manuals and reality is quite large for some cameras...

The second point isn't just the sky is it?

Comment (mwcraig) [reply notebooks/01-08-Overscan.ipynb]: > The second point isn't just the sky is it?

That is probably the case; I'm still a little bothered by how much the overscan differs between  different types of images on that camera.

**Pull request closed without merging**